.changelog/2505.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:new-datasource
+data-source/mongodbatlas_streamprocessors
```

Comment:
```suggestion
data-source/mongodbatlas_stream_processors
```
most importantly to adjust implementation as well if it does not use this name

Reply:
done in 8345bb61353b1580ffbd3b0cb4f412fa4acdb1a5